src/codegen/expression.rs

```diff
@@ -1383,6 +1400,14 @@ fn require(
                                     error_string.as_bytes().to_vec(),
                                 ),
                             ),
+                            (
+                                FormatArg::StringLiteral,
+                                Expression::BytesLiteral(
+                                    Loc::Codegen,
+                                    Type::Bytes(",\n".as_bytes().len() as u8),
+                                    ",\n".as_bytes().to_vec(),
+                                ),
+                            ),
```

Comment:
Might as well concatenate the last two string literals at compile time rather than runtime.